test/celeritas/mat/IsotopeSelector.test.cc

```diff
@@ -0,0 +1,178 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/mat/IsotopeSelector.test.cc
+//---------------------------------------------------------------------------//
+#include "celeritas/mat/IsotopeSelector.hh"
+
+#include <memory>
+#include <random>
+
+#include "celeritas/mat/MaterialParams.hh"
+#include "celeritas/random/SequenceEngine.hh"
+
+#include "celeritas_test.hh"
+
+namespace celeritas
+{
+namespace test
+{
+//---------------------------------------------------------------------------//
+using MaterialParamsRef = MaterialParams::HostRef;
+
+//---------------------------------------------------------------------------//
+// TEST HARNESS
+//---------------------------------------------------------------------------//
+
+class IsotopeSelectorTest : public Test
+{
+  public:
+    //!@{
+    //! \name Type aliases
+    using RandomEngine = std::mt19937;
+    //!@}
+
+  protected:
+    void SetUp() override
```

Comment:
Instead of copying these from the material test, create a simple test harness class shared between the two.